contrib/debian/fwupd.install

```diff
@@ -9,6 +9,7 @@ usr/share/polkit-1/*
 usr/share/locale
 usr/share/metainfo/*
 usr/libexec/fwupd/*
+usr/lib/modules-load.d/*.conf
```

Comment:
Unfortunately I realize this probably needs some packaging transitional code since we put something into `/etc` previously.  Don't worry about blocking the PR on it. i'll open a separate issue to get to it later.